katdal/chunkstore_s3.py

```diff
@@ -270,139 +282,60 @@ class S3ChunkStore(ChunkStore):
 
     Parameters
     ----------
-    session_factory : callable
-        A callable called with no arguments that returns an instance of
-        :class:`requests.Session`. The returned session is only used by
-        one thread at a time.
     url : str
-        Base URL for the S3 service. It can be specified as either bytes or
-        unicode, and is converted to the native string type with UTF-8.
+        Endpoint of S3 service, e.g. 'http://127.0.0.1:9000'. It can be
+        specified as either bytes or unicode, and is converted to the native
+        string type with UTF-8.
+    timeout : int or float, optional
```

Comment:
If you're documenting this as optional, then all the subsequent parameters should probably be documented as optional too.

Reply:
Done.